src/ultils/status.js

```diff
@@ -0,0 +1,26 @@
+import listItems from './data.js';
+import displayListItems from './template.js';
+
+displayListItems();
+
+const updateItems = () => {
+  document.querySelectorAll('.list-item .checked').forEach((item) => {
+    item.addEventListener('change', (e) => {
+      const data = JSON.parse(localStorage.getItem('listitems')) || listItems
+      const filtered = data.filter((item) => item.index !== parseInt(e.target.dataset.id, 10))
+      const currentData = data.find((item) => item.index === parseInt(e.target.dataset.id, 10))
+      currentData.completed = !currentData.completed;
+      filtered.push(currentData);
+      // localStorage.setItem('listitems', JSON.stringify(filtered));
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner

Reply:
This issue has been addressed well. Nice job! :pray: